Source/WebKit/NetworkProcess/NetworkSession.h

```diff
@@ -209,6 +209,7 @@ class NetworkSession : public WebCore::SWServerDelegate, public CanMakeCheckedPt
 
     WebCore::SWServer* swServer() { return m_swServer.get(); }
     WebCore::SWServer& ensureSWServer();
+    Ref<WebCore::SWServer> protectedSWServer();
```

Comment:
This should be named `ensureProtectedSWServer()` otherwise it is inconsistent with the 2 functions above.

Reply:
A good point, this name sounds like the function that returns a `RefPtr` and can return `nullptr`.